### PR TITLE
Added ability to install apt dependencies

### PR DESCRIPTION
**Pull request by andrew-kelly-neutralaiz**
### Description
I need the ability to add some additional libraries outside of python itself as I wanted to create a tool for running headless chrome.  That required a number of dependancies installed using the apt tool.  

### Related Issues

NA
### Solution and Design

I duplicated what is done for pip installs and modified it for apt.

### Test Plan

Include a file called /app/requirements_apt.txt, add the name of an apt package, monitor stack logs for successful installation of said package on docker compose up.

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update

### Checklist
- [X] My pull request is atomic and focuses on a single change.
- [X] I have read the contributing guide and my code conforms to the guidelines.
- [X] I have documented my changes clearly and comprehensively.
- [X] I have added the required tests.

### CODE REVIEW

File: tools.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
     "tools": {
+        "duck_duck_go": "https://github.com/TransformerOptimus/SuperAGI-Tools/tree/main/duck_duck_go",
+        "google_analytics": "https://github.com/TransformerOptimus/SuperAGI-Tools/tree/main/google_analytics",
+        "notion": "https://github.com/TransformerOptimus/SuperAGI-Tools/tree/main/notion",
+        "agent-manager": "https://github.com/NeutralAIz/agent-manager",
+        "llm-direct": "https://github.com/NeutralAIz/llm-direct"
     }
-}
+}
```
Comment (luciferlinx101) [inline tools.json:9]: Can you revert tools.json

File: docker-compose.yaml [hunk 2]
```diff
@@ -37,8 +37,8 @@ services:
     networks:
       - super_network
 #    uncomment to expose redis port to host
-#    ports:
-#      - "6379:6379"
+    ports:
+      - "6379:6379"
     volumes:
```
Comment (luciferlinx101) [inline docker-compose.yaml:42]: Revert these ports exposed, comment them back

File: docker-compose.yaml [hunk 3]
```diff
@@ -53,8 +53,8 @@ services:
     networks:
       - super_network
 #    uncomment to expose postgres port to host
-#    ports:
-#      - "5432:5432"
+    ports:
+      - "5432:5432"
```
Comment (luciferlinx101) [inline docker-compose.yaml:58]: Revert back and comment them back, change only required docs

File: superagi/helper/auth.py [hunk 4]
```diff
@@ -43,6 +44,20 @@ def get_user_organisation(Authorize: AuthJWT = Depends(check_auth)):
...
+    Args:
+        Authorize (AuthJWT, optional): Instance of AuthJWT class to authorize the user. Defaults to Depends on check_auth().
+
+    Returns:
+        Organisation: Instance of Organisation class to which the authenticated user belongs.
+    """
+    organisation = get_user_organisation(AuthJWT)
+    project = db.session.query(Project).filter(Project.organisation_id == organisation.id, Project.name == "Default Project").first()
+    return project
+
```
Comment (luciferlinx101) [inline superagi/helper/auth.py:61]: Can you explain how is this change relevant to your raised PR?

File: workspace/word_count.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+Google: 1, About: 1, Store: 1, Gmail: 1, Images: 1, Sign: 1, in: 1, See: 1, more: 1, Delete: 2, Report: 1, inappropriate: 1, predictions: 1, Google: 1, offered: 1, in: 1, Français: 1, Canada: 1, Advertising: 1, Business: 1, How: 1, Search: 6, works: 1, Privacy: 1, Terms: 1, Settings: 1, settings: 1, Advanced: 1, search: 1, Your: 1, data: 1, in: 1, Search: 1, Search: 1, history: 1, Search: 1, help: 1, Send: 1, feedback: 1, Dark: 1, theme: 1, Off: 1, Google: 1, apps: 1
```
Comment (luciferlinx101) [inline workspace/word_count.txt:1]: Can you remove these or explain why is it here?

File: workspace/word_counts.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+Google: 3, About: 1, Store: 1, Gmail: 1, Images: 1, Sign: 1, in: 1, See: 1, more: 1, Delete: 2, Report: 1, inappropriate: 1, predictions: 1, offered: 1, in: 1, Français: 1, Canada: 1, Advertising: 1, Business: 1, How: 1, Search: 4, works: 1, Privacy: 1, Terms: 1, Settings: 2, Advanced: 1, Your: 1, data: 1, history: 1, help: 1, Send: 1, feedback: 1, Dark: 1, theme: 1, Off: 1, apps: 1
```
Comment (luciferlinx101) [inline workspace/word_counts.txt:1]: Can you remove these or explain why is it here?

**Comment by luciferlinx101** [comment]
Also, a major problem is how can this be handled in MacOS and Windows, your solution seems just to be fine with Ubuntu.

**Comment by andrew-kelly-neutralaiz** [comment]
Created a new PR as this one became polluted with accidentally added commits.
